content/01_WhyRx.md

```diff
@@ -1,241 +1,126 @@
+- Reports from monitoring devices such as a flow sensor in a water utility's infrastructure, or the monitoring and diagnostic features in networking equipment
+- Location data from mobile systems such as [AIS](https://github.com/ais-dotnet/) messages from ships, or automotive telemetry
+- Operating system events such as filesystem activity, or WMI events
+- Road traffic information, such as notifications of accidents or changes in average speed
+- Integration with a [Complex Event Processing (CEP)](https://en.wikipedia.org/wiki/Complex_event_processing) engine
+- UI events such as mouse movement or button clicks
+
+Rx is also good way to model domain events—these may occur as a result of some of the events just described, but after processing them to produce events that more directly represent application concepts. These might include:
+
+- Property or state changes on domain objects such as "Order Status Updated", or "Registration Accepted"
+- Changes to collections of domain objects, such as "New Registration Created"
+
+Events might also represent insights derived from incoming events and historical data such as:
+
+- A broadband customer might have become an unwitting participant in a DDoS attack
+- [CNC](https://en.wikipedia.org/wiki/Numerical_control) [Milling Machine](https://en.wikipedia.org/wiki/Milling_(machining)) MFZH12's number 4 axis bearing is exhibiting signs of wear at a significantly higher rate than the nominal profile
+- If the user wants to arrive on time at their meeting half way across town, the current traffic conditions suggest they should leave in the next 10 minutes
+
+These three sets of examples show how applications might progressively increase the value of the information as they process events. We start with raw events, which we then enhance to produce domain-specific events, and we then perform analysis to produce notifications that the application's users will really care about. Each stage of processing increases the value of the messages that emerge. Each stage will typically also reduce the volume of messages—if we presented the raw events in the first category directly to users, they might be overwhelmed by the volume of messages, making it impossible to spot the important events. But if we only present them with notifications when our processing has detected something important, this will enable them to work more efficiently and accurately.
+
+The [`System.Reactive` library](https://www.nuget.org/packages/System.Reactive) provides tools for building exactly this kind of value-adding process, in which we tame high-volume raw event sources to produce high-value live insights. It provides a suite of operators that enable our code to express this kind of processing declaratively, as you'll see in subsequent chapters.
```

Comment:
We haven't really explained the link between Reactive Extensions / Rx.NET / Rx and System.Reactive.